src/hooks/useColorGenerator/useColorGenerator.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable valid-jsdoc */
```

Comment:
Let's use correct jsdoc, see the [reference](https://jsdoc.app/tags-example)

Reply:
similar approach to the documentation in used in the file [https://github.com/gravity-ui/uikit/blob/main/src/hooks/useFileInput/useFileInput.ts](url)

Reply:
That's also incorrect and should be fixed